src/test/serialization_tests.cpp

```diff
@@ -366,6 +368,31 @@ TEST_P(SerializeKFSInferOutputTensorNegative, SerializeTensorProtoShouldSucceedF
         << "should fail";
 }
 
+TEST(SerializeKFSGRPCPredictResponse, ShouldSuccessForSupportedPrecision) {
```

Comment:
Shouldn't this be parametrized?

Reply:
SerializeTensor tests are parametrized. For whole predict this should suffice.